docs/source/ccvm_equations_of_motion.rst

```diff
@@ -0,0 +1,44 @@
+
+CCVM Simulator - Equations of Motion
+====================================
+
+Architecture of Coherent Continuous Variable Machine (CCVM)
+-----------------------------------------------------------
+
+.. figure:: _static/images/ccvm-architecture.png
+    :align: center
+    :width: 50%
+    :alt: CCVM Architecture
+
+    Schematic of the CCVM architecture.
+
+This figure shows the schematic of the architecture of a coherent continuous-variable machine (CCVM), as described in Farhad et al 2022. Squeezed coherent states of light are generated and amplified using an optical parametric amplifier (OPA) that includes a nonlinear crystal pumped by a laser that, together with a ring cavity, constitutes a degenerate optical parametric oscillator (DOPO). Information is encoded in the amplitudes of the time-multiplexed optical pulses that propagate within the resonator, which are coherently amplified each time they pass through the OPA element. Coupling between each pair of optical pulses is implemented by taking a portion of each pulse using an output coupler, calculating the gradient of the objective function using either a delay-line network or a measurement-feedback scheme, and feeding the result into each optical pulse using an injection coupler. These couplers, in general, can be variable beamsplitters. Using this scheme, and repeating this process an arbitrary number of times, the system is driven towards the minimum of the objective function, whose gradient is implemented using the coupling mechanism. The solution of the optimization problem is represented by the final values of the amplitudes of the optical pulses in the cavity.
+
+
+Parameter values
+----------------
+
+These parameters, chosen for the three types of solvers implemented within the simulator, are presented in the following table:
```

Comment:
I think the table describes clearly the relevant parameters for Langeving equation, DL-CCVM and MF-CCVM.
Therefore, in my opinion it might be redundant to describe them repeatedly.

Reply:
I took a closer look and realized that the next few pages describe what the parameters mean, e.g. σ is described in the Langevin page to be the diffusion term. 
I agree, defining them twice would be redundant. How about I update this page with a note to let readers know to read ahead for definitions?

However, I still don't see anywhere that describes _how_ we chose these values. Why is 0.5 the value we use for σ? Were they empirically determined or is there some science behind it?

Reply:
- Regarding the first question, yes that will be helpful.
- Before responding to next question, I would say the potential users were supposed to have prerequisite knowledge about coherent Ising machine (CIM) and CCVM systems. If not the curious users were referred in the text to Farhad et al 2022 and the references in.
Consequently, I think why and how to choose any specific value for a specific parameter is beyond the scope of this documentation.

Reply:
"Khosravi et al. (2022)"

Reply:
Thanks @MehmetC-1QBit . Sure, that makes sense. I'll just point them to the correct resources, then.
And thanks @MarkoB4t1qbit , I will fix that, too.